SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -14,6 +25,25 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
 
     override fun onViewTaken(view: ApplicationContract.View) {
         this.view = view
-        view.setLabel(createApplicationScreenMessage())
+        view.setLabel(createApplicationScreenMessage()+"!!!!")
+        launch { getStationList() }
```

Comment:
I think you could move launch to within getStationList (and then make getStationList no longer a suspend function)